python/GafferSceneUI/SetFilterUI.py

```diff
@@ -57,15 +57,22 @@
 
 			"description",
 			"""
-			The name of a set that defines the locations to
-			be matched.
+			A set expression that computes a set that defines
```

Comment:
Could you include a simple example of an expression here, and mention that a kind soul has also provided extensive documentation of the syntax in the reference section?